### ISSUE TITLE
RB: CI variables doesn't set

### ISSUE THREAD

**Issue by BarsukovNik** [janbarari/gradle-analytics-plugin#89]
Hello! When I am using this plugin locally everything is fine but when I try to set it on CI I noticed a bug - the some variables of the ci config didn't set. 
```
ci = mysql {
            host = "MYSQL_DATABASE_HOST"
            name = "MYSQL_DATABASE_NAME"
            user = "MYSQL_DATABASE_USER"
            password = "MYSQL_DATABASE_PASSWORD"
            port = MYSQL_DATABASE_PORT // Default is 3306
        }
```
I fixed it like 
```
ci = mysql {
            host = "MYSQL_DATABASE_HOST"
            name = "MYSQL_DATABASE_NAME"
            user = "MYSQL_DATABASE_USER"
            password = "MYSQL_DATABASE_PASSWORD"
            port = MYSQL_DATABASE_PORT // Default is 3306
        }.apply{
            user = "MYSQL_DATABASE_USER"
            password = "MYSQL_DATABASE_PASSWORD"
        }
```
And now it works

**Comment by janbarari** [comment]
Hey @BarsukovNik 
Thank you for reporting this issue

Can you tell me what CI you use? That would help me for reproducing the issue.

**Comment by BarsukovNik** [comment]
@janbarari I'm using gitlab

**Comment by janbarari** [comment]
Hmm. So, I need to reproduce it on Gitlab CI.

**Comment by janbarari** [comment]
Thank you so much for reporting this bug 🧡

**Comment by janbarari** [comment]
Added in milestone `1.0.0-beta6`. will be available in `v1.0.0-beta6` in 1 or 2 weeks.
